### ISSUE TITLE
.screen-reader-text CSS update for latest consistency

### ISSUE THREAD

**Issue by colorful-tones** [WordPress/gutenberg#65954]
## What problem does this address?

A follow-up to #65409 - It was decided that the CSS for standard `.screen-reader-text` should be updated across many surface areas of WordPress, Gutenberg, and WordPress Accessibility Handbook to reflect the latest CSS:

```
    border: 0;
    clip-path: inset(50%);
    height: 1px;
    margin: -1px;
    overflow: hidden;
    padding: 0;
    position: absolute;
    width: 1px;
    word-wrap: normal !important;
```

This issue and related pull requests should only address code in the Gutenberg project, and WordPress Make Handbook / Trac items will be addressed separately. (see: https://github.com/WordPress/gutenberg/pull/65409#issuecomment-2400422697)

When performing a search for `inset(50%)` the Gutenberg project offers these areas that may need to be addressed: https://github.com/search?q=repo%3AWordPress%2Fgutenberg%20inset(50%25)&type=code

## What is your proposed solution?

Review and update the following files if they're attempting to provide a `.screen-reader-text`-like CSS solution.

- [x] [packages/block-library/src/common.scss](https://github.com/WordPress/gutenberg/blob/b15b8fd522109ae96a26fb869ed663bcf6b39bf7/packages/block-library/src/common.scss#L80)
- [x] [packages/block-editor/src/components/responsive-block-control/style.scss](https://github.com/WordPress/gutenberg/blob/b15b8fd522109ae96a26fb869ed663bcf6b39bf7/packages/block-editor/src/components/responsive-block-control/style.scss#L4)
- [ ] [lock-directory/src/components/downloadable-block-list-item/style.scss](https://github.com/WordPress/gutenberg/blob/b15b8fd522109ae96a26fb869ed663bcf6b39bf7/packages/block-directory/src/components/downloadable-block-list-item/style.scss#L44)

**Comment by carolinan** [comment]
Hi
It looks like all items on the to do list have been solved? Can this be closed?
